src/dawn/engine_dawn.h

```diff
@@ -29,6 +29,10 @@
 namespace amber {
 namespace dawn {
 
+static constexpr uint32_t kMaxColorAttachments = 4u;
```

Comment:
constexpr doesn't work in VS2013.  I'm not sure if we care to support it or not.

If we need it, an alternative is

enum : uint32_t { kMaxColorAttachments = 4u, kMaxVertexInputs = 16u, kMaxVertexAttributes = 16u };

Reply:
No it's not needed I replaced it with const and moved them to the anonymes space in engine_dawn.cc.